core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.component.html

```diff
@@ -88,67 +88,67 @@
   </div>
 
   <ul>
-    <li ng-class="{'active': $ctrl.currentUrl === '/community-library'}">
+    <li [ngClass]="{'active': currentUrl === '/community-library'}">
```

Comment:
@safwan-1998 Can you please try removing `side-navigation-bar` from the css selectors above and test? I think that should work although that should not be the solution i.e, you have to specify more precise selectors.

Reply:
I tried by removing `side-navigation-bar` from selectors still didn't got the result I looked others file also they too don't have `component selectors` with `css selectors`.

Reply:
Alternatively, you can wrap the entire component in a div of class `side-navigation-bar` and change the rules in the following way:
Before: `side-navigation-bar .some-class`
After: `.side-navigation-bar .some-class` (Notice the extra dot before side-navigation-bar in after)

Reply:
@srijanreddy98 actually I did that already but still not working. maybe there is some problem with CSS because when I used the same style in the top navigation bar it worked but it is not working with side-navigation-bar. I think there must be some problem with the oppia.css file.